### PR TITLE
More work on the Docs

### PR DESCRIPTION
**Pull request by adswa**
ready for review

### CODE REVIEW

File: docs/source/getting-started.rst [hunk 1]
```diff
@@ -57,4 +66,19 @@ The two suites you will always be able to select between is a "simplified" comma
 DataLad extensions can add additional suites when you install them.
 Please note that we recommend the "simplified" command suite to users, as the complete suite can contain experimental implementations.
 
-.. image:: _static/suite_menu.png
\ No newline at end of file
+.. image:: _static/suite_menu.png
+
+The Utilities and Help Tab
```
Comment (mih) [inline docs/source/getting-started.rst:71]: Here and below "tab" should be "menu", I think

File: docs/source/index.rst [hunk 2]
```diff
@@ -18,6 +19,13 @@ Current functionality supported via the simplified suite includes:
 * `save`_ the state of a dataset
 * `update`_ from a sibling
 
+What DataLad Gooey is not
+=========================
+DataLad Gooey has a number of cool features, but here are features that you will need to use other tools for:
+
+* An interface to visualize revision histories of DataLad datasets. Please refer to many of the available visual Git visualization tools
+* An interface for advanced Git operations such as branching, resetting, reverting, or otherwise interacting with commit history. Please refer to your favourite Git client or the command line for these operations.
+
```
Comment (mih) [inline docs/source/index.rst:28]: Maybe say that

```suggestion
  DataLad Gooey will detect such external operations, and will update its view accordingly.
```

**Review by mih — CHANGES_REQUESTED** [review]
Thx!

I have one suggestion, and one thing that is likely an error.

**Review by jsheunis — APPROVED** [review]
All looks great, I like the additional structure. I made a small PR to your branch with some minor edits to headings

File: docs/source/getting-started.rst (hunk 1 above)
Comment (adswa) [reply docs/source/getting-started.rst]: done

**Pull request merged**